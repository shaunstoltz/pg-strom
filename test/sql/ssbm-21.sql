SET pg_strom.regression_test_mode = on;
SET search_path = pgstrom_regress,public;
SET pg_strom.debug_kernel_source = off;
--Q2_1
SET pg_strom.enabled = on;
select sum(lo_revenue), d_year, p_brand1
from lineorder, date1, part, supplier
where lo_orderdate = d_datekey
and lo_partkey = p_partkey
and lo_suppkey = s_suppkey
and p_category = 'MFGR#12'
and s_region = 'AMERICA'
  group by d_year, p_brand1
  order by d_year, p_brand1;

SET pg_strom.enabled = off;
select sum(lo_revenue), d_year, p_brand1
from lineorder, date1, part, supplier
where lo_orderdate = d_datekey
and lo_partkey = p_partkey
and lo_suppkey = s_suppkey
and p_category = 'MFGR#12'
and s_region = 'AMERICA'
  group by d_year, p_brand1
  order by d_year, p_brand1;
