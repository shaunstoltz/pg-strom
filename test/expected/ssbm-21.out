SET pg_strom.regression_test_mode = on;
SET search_path = pgstrom_regress,public;
SET pg_strom.debug_kernel_source = off;
--Q2_1
SET pg_strom.enabled = on;
select sum(lo_revenue), d_year, p_brand1
from lineorder, date1, part, supplier
where lo_orderdate = d_datekey
and lo_partkey = p_partkey
and lo_suppkey = s_suppkey
and p_category = 'MFGR#12'
and s_region = 'AMERICA'
  group by d_year, p_brand1
  order by d_year, p_brand1;
    sum     | d_year | p_brand1  
------------+--------+-----------
 1260895183 |   1992 | MFGR#121 
 1529040688 |   1992 | MFGR#1210
 1316620329 |   1992 | MFGR#1211
 1264923920 |   1992 | MFGR#1212
 1285225081 |   1992 | MFGR#1213
 1373572833 |   1992 | MFGR#1214
 1417254666 |   1992 | MFGR#1215
 1435737591 |   1992 | MFGR#1216
 1477213665 |   1992 | MFGR#1217
 1305152761 |   1992 | MFGR#1218
 1220246424 |   1992 | MFGR#1219
 1560431975 |   1992 | MFGR#122 
 1335277644 |   1992 | MFGR#1220
 1180863789 |   1992 | MFGR#1221
 1339890711 |   1992 | MFGR#1222
 1191488530 |   1992 | MFGR#1223
 1586901983 |   1992 | MFGR#1224
 1361272573 |   1992 | MFGR#1225
 1363427683 |   1992 | MFGR#1226
 1159034853 |   1992 | MFGR#1227
 1303840352 |   1992 | MFGR#1228
 1365561884 |   1992 | MFGR#1229
 1499718114 |   1992 | MFGR#123 
 1266508120 |   1992 | MFGR#1230
 1172869349 |   1992 | MFGR#1231
 1417770256 |   1992 | MFGR#1232
 1307132386 |   1992 | MFGR#1233
 1383010567 |   1992 | MFGR#1234
 1319571290 |   1992 | MFGR#1235
 1278118865 |   1992 | MFGR#1236
 1357687347 |   1992 | MFGR#1237
 1213722223 |   1992 | MFGR#1238
 1243514576 |   1992 | MFGR#1239
 1148570675 |   1992 | MFGR#124 
 1438039743 |   1992 | MFGR#1240
 1045291411 |   1992 | MFGR#125 
 1267131969 |   1992 | MFGR#126 
 1488259946 |   1992 | MFGR#127 
 1409242662 |   1992 | MFGR#128 
 1289288303 |   1992 | MFGR#129 
 1390344799 |   1993 | MFGR#121 
 1280596203 |   1993 | MFGR#1210
 1307091945 |   1993 | MFGR#1211
 1363147844 |   1993 | MFGR#1212
 1144515341 |   1993 | MFGR#1213
 1193584943 |   1993 | MFGR#1214
 1318096658 |   1993 | MFGR#1215
 1386837857 |   1993 | MFGR#1216
 1436780606 |   1993 | MFGR#1217
 1273834783 |   1993 | MFGR#1218
 1160827650 |   1993 | MFGR#1219
 1323633796 |   1993 | MFGR#122 
 1278083255 |   1993 | MFGR#1220
 1224640581 |   1993 | MFGR#1221
 1345861271 |   1993 | MFGR#1222
 1323758917 |   1993 | MFGR#1223
 1597294073 |   1993 | MFGR#1224
 1245844239 |   1993 | MFGR#1225
 1410204133 |   1993 | MFGR#1226
 1198412751 |   1993 | MFGR#1227
 1453087368 |   1993 | MFGR#1228
 1234792711 |   1993 | MFGR#1229
 1315478166 |   1993 | MFGR#123 
 1182390132 |   1993 | MFGR#1230
 1251553076 |   1993 | MFGR#1231
 1394096170 |   1993 | MFGR#1232
 1247418659 |   1993 | MFGR#1233
 1436428112 |   1993 | MFGR#1234
 1311679523 |   1993 | MFGR#1235
 1368777581 |   1993 | MFGR#1236
 1177424523 |   1993 | MFGR#1237
 1016919631 |   1993 | MFGR#1238
 1374656436 |   1993 | MFGR#1239
 1403655022 |   1993 | MFGR#124 
 1465101536 |   1993 | MFGR#1240
 1310854959 |   1993 | MFGR#125 
 1203937239 |   1993 | MFGR#126 
 1465967720 |   1993 | MFGR#127 
 1378379694 |   1993 | MFGR#128 
 1192522155 |   1993 | MFGR#129 
 1416379585 |   1994 | MFGR#121 
 1255931227 |   1994 | MFGR#1210
 1400933913 |   1994 | MFGR#1211
 1428589263 |   1994 | MFGR#1212
 1244869343 |   1994 | MFGR#1213
 1206025728 |   1994 | MFGR#1214
 1427082788 |   1994 | MFGR#1215
 1424231833 |   1994 | MFGR#1216
 1618539964 |   1994 | MFGR#1217
 1444587606 |   1994 | MFGR#1218
 1077002455 |   1994 | MFGR#1219
 1544557366 |   1994 | MFGR#122 
 1241514336 |   1994 | MFGR#1220
 1187357573 |   1994 | MFGR#1221
 1331495613 |   1994 | MFGR#1222
 1299839995 |   1994 | MFGR#1223
 1577278969 |   1994 | MFGR#1224
 1270551718 |   1994 | MFGR#1225
 1472667041 |   1994 | MFGR#1226
 1270280584 |   1994 | MFGR#1227
 1263225465 |   1994 | MFGR#1228
 1444859099 |   1994 | MFGR#1229
 1398778065 |   1994 | MFGR#123 
 1276884516 |   1994 | MFGR#1230
 1219855989 |   1994 | MFGR#1231
 1485271052 |   1994 | MFGR#1232
 1315491728 |   1994 | MFGR#1233
 1476307414 |   1994 | MFGR#1234
 1411355243 |   1994 | MFGR#1235
 1343930323 |   1994 | MFGR#1236
 1228823454 |   1994 | MFGR#1237
 1048226295 |   1994 | MFGR#1238
 1249124093 |   1994 | MFGR#1239
 1471939775 |   1994 | MFGR#124 
 1435125166 |   1994 | MFGR#1240
 1134577269 |   1994 | MFGR#125 
 1091160011 |   1994 | MFGR#126 
 1235776041 |   1994 | MFGR#127 
 1249382981 |   1994 | MFGR#128 
 1171691495 |   1994 | MFGR#129 
 1471530793 |   1995 | MFGR#121 
 1411298960 |   1995 | MFGR#1210
 1333836977 |   1995 | MFGR#1211
 1464227104 |   1995 | MFGR#1212
 1403461088 |   1995 | MFGR#1213
 1299365156 |   1995 | MFGR#1214
 1483941877 |   1995 | MFGR#1215
 1518323135 |   1995 | MFGR#1216
 1596644757 |   1995 | MFGR#1217
 1329166399 |   1995 | MFGR#1218
 1212265247 |   1995 | MFGR#1219
 1547712869 |   1995 | MFGR#122 
 1181007044 |   1995 | MFGR#1220
 1236209682 |   1995 | MFGR#1221
 1300419312 |   1995 | MFGR#1222
 1217269166 |   1995 | MFGR#1223
 1564003020 |   1995 | MFGR#1224
 1193131787 |   1995 | MFGR#1225
 1385825544 |   1995 | MFGR#1226
 1178579905 |   1995 | MFGR#1227
 1273612007 |   1995 | MFGR#1228
 1323867284 |   1995 | MFGR#1229
 1326044250 |   1995 | MFGR#123 
 1311793532 |   1995 | MFGR#1230
 1177831833 |   1995 | MFGR#1231
 1577494964 |   1995 | MFGR#1232
 1331663072 |   1995 | MFGR#1233
 1509856419 |   1995 | MFGR#1234
 1400750393 |   1995 | MFGR#1235
 1258902627 |   1995 | MFGR#1236
 1349275325 |   1995 | MFGR#1237
  995506714 |   1995 | MFGR#1238
 1432735312 |   1995 | MFGR#1239
 1329328759 |   1995 | MFGR#124 
 1651872286 |   1995 | MFGR#1240
 1076333113 |   1995 | MFGR#125 
 1189955473 |   1995 | MFGR#126 
 1244513498 |   1995 | MFGR#127 
 1406475452 |   1995 | MFGR#128 
 1278945677 |   1995 | MFGR#129 
 1259642958 |   1996 | MFGR#121 
 1422904368 |   1996 | MFGR#1210
 1363334595 |   1996 | MFGR#1211
 1460014204 |   1996 | MFGR#1212
 1305010923 |   1996 | MFGR#1213
 1280222971 |   1996 | MFGR#1214
 1286892297 |   1996 | MFGR#1215
 1416073184 |   1996 | MFGR#1216
 1536725126 |   1996 | MFGR#1217
 1367817711 |   1996 | MFGR#1218
 1224457367 |   1996 | MFGR#1219
 1481925649 |   1996 | MFGR#122 
 1281005021 |   1996 | MFGR#1220
 1240637787 |   1996 | MFGR#1221
 1463484958 |   1996 | MFGR#1222
 1158640751 |   1996 | MFGR#1223
 1472635321 |   1996 | MFGR#1224
 1253877465 |   1996 | MFGR#1225
 1429974796 |   1996 | MFGR#1226
 1223357966 |   1996 | MFGR#1227
 1392629765 |   1996 | MFGR#1228
 1322562055 |   1996 | MFGR#1229
 1390631915 |   1996 | MFGR#123 
 1348542024 |   1996 | MFGR#1230
 1183428508 |   1996 | MFGR#1231
 1460204214 |   1996 | MFGR#1232
 1310832108 |   1996 | MFGR#1233
 1534596742 |   1996 | MFGR#1234
 1241281621 |   1996 | MFGR#1235
 1419622684 |   1996 | MFGR#1236
 1163085173 |   1996 | MFGR#1237
 1063788367 |   1996 | MFGR#1238
 1199194189 |   1996 | MFGR#1239
 1346687277 |   1996 | MFGR#124 
 1571760910 |   1996 | MFGR#1240
 1101097053 |   1996 | MFGR#125 
 1090122177 |   1996 | MFGR#126 
 1544330235 |   1996 | MFGR#127 
 1370946911 |   1996 | MFGR#128 
 1269919349 |   1996 | MFGR#129 
 1247033703 |   1997 | MFGR#121 
 1347066699 |   1997 | MFGR#1210
 1455475847 |   1997 | MFGR#1211
 1399803223 |   1997 | MFGR#1212
 1122484545 |   1997 | MFGR#1213
 1337347144 |   1997 | MFGR#1214
 1402515447 |   1997 | MFGR#1215
 1420052026 |   1997 | MFGR#1216
 1495082264 |   1997 | MFGR#1217
 1357824652 |   1997 | MFGR#1218
 1127355577 |   1997 | MFGR#1219
 1374061472 |   1997 | MFGR#122 
 1218324236 |   1997 | MFGR#1220
 1389236025 |   1997 | MFGR#1221
 1414147439 |   1997 | MFGR#1222
 1281336362 |   1997 | MFGR#1223
 1523832297 |   1997 | MFGR#1224
 1345236090 |   1997 | MFGR#1225
 1523314702 |   1997 | MFGR#1226
 1158340061 |   1997 | MFGR#1227
 1410752422 |   1997 | MFGR#1228
 1397754873 |   1997 | MFGR#1229
 1575447477 |   1997 | MFGR#123 
 1225023526 |   1997 | MFGR#1230
 1113962713 |   1997 | MFGR#1231
 1511260579 |   1997 | MFGR#1232
 1261364847 |   1997 | MFGR#1233
 1555658369 |   1997 | MFGR#1234
 1325601283 |   1997 | MFGR#1235
 1288704598 |   1997 | MFGR#1236
 1215812057 |   1997 | MFGR#1237
 1076279861 |   1997 | MFGR#1238
 1277424841 |   1997 | MFGR#1239
 1137540388 |   1997 | MFGR#124 
 1562295688 |   1997 | MFGR#1240
 1324107868 |   1997 | MFGR#125 
 1304429364 |   1997 | MFGR#126 
 1495219935 |   1997 | MFGR#127 
 1435838994 |   1997 | MFGR#128 
 1380654142 |   1997 | MFGR#129 
  728405195 |   1998 | MFGR#121 
  772241518 |   1998 | MFGR#1210
  654604431 |   1998 | MFGR#1211
  890208493 |   1998 | MFGR#1212
  640003860 |   1998 | MFGR#1213
  748112862 |   1998 | MFGR#1214
  737206930 |   1998 | MFGR#1215
  879344897 |   1998 | MFGR#1216
  797370310 |   1998 | MFGR#1217
  795668107 |   1998 | MFGR#1218
  660754584 |   1998 | MFGR#1219
  759924147 |   1998 | MFGR#122 
  776200672 |   1998 | MFGR#1220
  759235159 |   1998 | MFGR#1221
  858128275 |   1998 | MFGR#1222
  772868837 |   1998 | MFGR#1223
  827964147 |   1998 | MFGR#1224
  772631022 |   1998 | MFGR#1225
  934347224 |   1998 | MFGR#1226
  704757699 |   1998 | MFGR#1227
  719700730 |   1998 | MFGR#1228
  844398174 |   1998 | MFGR#1229
  741350529 |   1998 | MFGR#123 
  697727449 |   1998 | MFGR#1230
  715848717 |   1998 | MFGR#1231
  872721297 |   1998 | MFGR#1232
  698644386 |   1998 | MFGR#1233
  786058527 |   1998 | MFGR#1234
  711299651 |   1998 | MFGR#1235
  764901787 |   1998 | MFGR#1236
  689507081 |   1998 | MFGR#1237
  618842999 |   1998 | MFGR#1238
  854888093 |   1998 | MFGR#1239
  662699900 |   1998 | MFGR#124 
  937491980 |   1998 | MFGR#1240
  654370332 |   1998 | MFGR#125 
  798795601 |   1998 | MFGR#126 
  832030830 |   1998 | MFGR#127 
  787493701 |   1998 | MFGR#128 
  669040069 |   1998 | MFGR#129 
(280 rows)

SET pg_strom.enabled = off;
select sum(lo_revenue), d_year, p_brand1
from lineorder, date1, part, supplier
where lo_orderdate = d_datekey
and lo_partkey = p_partkey
and lo_suppkey = s_suppkey
and p_category = 'MFGR#12'
and s_region = 'AMERICA'
  group by d_year, p_brand1
  order by d_year, p_brand1;
    sum     | d_year | p_brand1  
------------+--------+-----------
 1260895183 |   1992 | MFGR#121 
 1529040688 |   1992 | MFGR#1210
 1316620329 |   1992 | MFGR#1211
 1264923920 |   1992 | MFGR#1212
 1285225081 |   1992 | MFGR#1213
 1373572833 |   1992 | MFGR#1214
 1417254666 |   1992 | MFGR#1215
 1435737591 |   1992 | MFGR#1216
 1477213665 |   1992 | MFGR#1217
 1305152761 |   1992 | MFGR#1218
 1220246424 |   1992 | MFGR#1219
 1560431975 |   1992 | MFGR#122 
 1335277644 |   1992 | MFGR#1220
 1180863789 |   1992 | MFGR#1221
 1339890711 |   1992 | MFGR#1222
 1191488530 |   1992 | MFGR#1223
 1586901983 |   1992 | MFGR#1224
 1361272573 |   1992 | MFGR#1225
 1363427683 |   1992 | MFGR#1226
 1159034853 |   1992 | MFGR#1227
 1303840352 |   1992 | MFGR#1228
 1365561884 |   1992 | MFGR#1229
 1499718114 |   1992 | MFGR#123 
 1266508120 |   1992 | MFGR#1230
 1172869349 |   1992 | MFGR#1231
 1417770256 |   1992 | MFGR#1232
 1307132386 |   1992 | MFGR#1233
 1383010567 |   1992 | MFGR#1234
 1319571290 |   1992 | MFGR#1235
 1278118865 |   1992 | MFGR#1236
 1357687347 |   1992 | MFGR#1237
 1213722223 |   1992 | MFGR#1238
 1243514576 |   1992 | MFGR#1239
 1148570675 |   1992 | MFGR#124 
 1438039743 |   1992 | MFGR#1240
 1045291411 |   1992 | MFGR#125 
 1267131969 |   1992 | MFGR#126 
 1488259946 |   1992 | MFGR#127 
 1409242662 |   1992 | MFGR#128 
 1289288303 |   1992 | MFGR#129 
 1390344799 |   1993 | MFGR#121 
 1280596203 |   1993 | MFGR#1210
 1307091945 |   1993 | MFGR#1211
 1363147844 |   1993 | MFGR#1212
 1144515341 |   1993 | MFGR#1213
 1193584943 |   1993 | MFGR#1214
 1318096658 |   1993 | MFGR#1215
 1386837857 |   1993 | MFGR#1216
 1436780606 |   1993 | MFGR#1217
 1273834783 |   1993 | MFGR#1218
 1160827650 |   1993 | MFGR#1219
 1323633796 |   1993 | MFGR#122 
 1278083255 |   1993 | MFGR#1220
 1224640581 |   1993 | MFGR#1221
 1345861271 |   1993 | MFGR#1222
 1323758917 |   1993 | MFGR#1223
 1597294073 |   1993 | MFGR#1224
 1245844239 |   1993 | MFGR#1225
 1410204133 |   1993 | MFGR#1226
 1198412751 |   1993 | MFGR#1227
 1453087368 |   1993 | MFGR#1228
 1234792711 |   1993 | MFGR#1229
 1315478166 |   1993 | MFGR#123 
 1182390132 |   1993 | MFGR#1230
 1251553076 |   1993 | MFGR#1231
 1394096170 |   1993 | MFGR#1232
 1247418659 |   1993 | MFGR#1233
 1436428112 |   1993 | MFGR#1234
 1311679523 |   1993 | MFGR#1235
 1368777581 |   1993 | MFGR#1236
 1177424523 |   1993 | MFGR#1237
 1016919631 |   1993 | MFGR#1238
 1374656436 |   1993 | MFGR#1239
 1403655022 |   1993 | MFGR#124 
 1465101536 |   1993 | MFGR#1240
 1310854959 |   1993 | MFGR#125 
 1203937239 |   1993 | MFGR#126 
 1465967720 |   1993 | MFGR#127 
 1378379694 |   1993 | MFGR#128 
 1192522155 |   1993 | MFGR#129 
 1416379585 |   1994 | MFGR#121 
 1255931227 |   1994 | MFGR#1210
 1400933913 |   1994 | MFGR#1211
 1428589263 |   1994 | MFGR#1212
 1244869343 |   1994 | MFGR#1213
 1206025728 |   1994 | MFGR#1214
 1427082788 |   1994 | MFGR#1215
 1424231833 |   1994 | MFGR#1216
 1618539964 |   1994 | MFGR#1217
 1444587606 |   1994 | MFGR#1218
 1077002455 |   1994 | MFGR#1219
 1544557366 |   1994 | MFGR#122 
 1241514336 |   1994 | MFGR#1220
 1187357573 |   1994 | MFGR#1221
 1331495613 |   1994 | MFGR#1222
 1299839995 |   1994 | MFGR#1223
 1577278969 |   1994 | MFGR#1224
 1270551718 |   1994 | MFGR#1225
 1472667041 |   1994 | MFGR#1226
 1270280584 |   1994 | MFGR#1227
 1263225465 |   1994 | MFGR#1228
 1444859099 |   1994 | MFGR#1229
 1398778065 |   1994 | MFGR#123 
 1276884516 |   1994 | MFGR#1230
 1219855989 |   1994 | MFGR#1231
 1485271052 |   1994 | MFGR#1232
 1315491728 |   1994 | MFGR#1233
 1476307414 |   1994 | MFGR#1234
 1411355243 |   1994 | MFGR#1235
 1343930323 |   1994 | MFGR#1236
 1228823454 |   1994 | MFGR#1237
 1048226295 |   1994 | MFGR#1238
 1249124093 |   1994 | MFGR#1239
 1471939775 |   1994 | MFGR#124 
 1435125166 |   1994 | MFGR#1240
 1134577269 |   1994 | MFGR#125 
 1091160011 |   1994 | MFGR#126 
 1235776041 |   1994 | MFGR#127 
 1249382981 |   1994 | MFGR#128 
 1171691495 |   1994 | MFGR#129 
 1471530793 |   1995 | MFGR#121 
 1411298960 |   1995 | MFGR#1210
 1333836977 |   1995 | MFGR#1211
 1464227104 |   1995 | MFGR#1212
 1403461088 |   1995 | MFGR#1213
 1299365156 |   1995 | MFGR#1214
 1483941877 |   1995 | MFGR#1215
 1518323135 |   1995 | MFGR#1216
 1596644757 |   1995 | MFGR#1217
 1329166399 |   1995 | MFGR#1218
 1212265247 |   1995 | MFGR#1219
 1547712869 |   1995 | MFGR#122 
 1181007044 |   1995 | MFGR#1220
 1236209682 |   1995 | MFGR#1221
 1300419312 |   1995 | MFGR#1222
 1217269166 |   1995 | MFGR#1223
 1564003020 |   1995 | MFGR#1224
 1193131787 |   1995 | MFGR#1225
 1385825544 |   1995 | MFGR#1226
 1178579905 |   1995 | MFGR#1227
 1273612007 |   1995 | MFGR#1228
 1323867284 |   1995 | MFGR#1229
 1326044250 |   1995 | MFGR#123 
 1311793532 |   1995 | MFGR#1230
 1177831833 |   1995 | MFGR#1231
 1577494964 |   1995 | MFGR#1232
 1331663072 |   1995 | MFGR#1233
 1509856419 |   1995 | MFGR#1234
 1400750393 |   1995 | MFGR#1235
 1258902627 |   1995 | MFGR#1236
 1349275325 |   1995 | MFGR#1237
  995506714 |   1995 | MFGR#1238
 1432735312 |   1995 | MFGR#1239
 1329328759 |   1995 | MFGR#124 
 1651872286 |   1995 | MFGR#1240
 1076333113 |   1995 | MFGR#125 
 1189955473 |   1995 | MFGR#126 
 1244513498 |   1995 | MFGR#127 
 1406475452 |   1995 | MFGR#128 
 1278945677 |   1995 | MFGR#129 
 1259642958 |   1996 | MFGR#121 
 1422904368 |   1996 | MFGR#1210
 1363334595 |   1996 | MFGR#1211
 1460014204 |   1996 | MFGR#1212
 1305010923 |   1996 | MFGR#1213
 1280222971 |   1996 | MFGR#1214
 1286892297 |   1996 | MFGR#1215
 1416073184 |   1996 | MFGR#1216
 1536725126 |   1996 | MFGR#1217
 1367817711 |   1996 | MFGR#1218
 1224457367 |   1996 | MFGR#1219
 1481925649 |   1996 | MFGR#122 
 1281005021 |   1996 | MFGR#1220
 1240637787 |   1996 | MFGR#1221
 1463484958 |   1996 | MFGR#1222
 1158640751 |   1996 | MFGR#1223
 1472635321 |   1996 | MFGR#1224
 1253877465 |   1996 | MFGR#1225
 1429974796 |   1996 | MFGR#1226
 1223357966 |   1996 | MFGR#1227
 1392629765 |   1996 | MFGR#1228
 1322562055 |   1996 | MFGR#1229
 1390631915 |   1996 | MFGR#123 
 1348542024 |   1996 | MFGR#1230
 1183428508 |   1996 | MFGR#1231
 1460204214 |   1996 | MFGR#1232
 1310832108 |   1996 | MFGR#1233
 1534596742 |   1996 | MFGR#1234
 1241281621 |   1996 | MFGR#1235
 1419622684 |   1996 | MFGR#1236
 1163085173 |   1996 | MFGR#1237
 1063788367 |   1996 | MFGR#1238
 1199194189 |   1996 | MFGR#1239
 1346687277 |   1996 | MFGR#124 
 1571760910 |   1996 | MFGR#1240
 1101097053 |   1996 | MFGR#125 
 1090122177 |   1996 | MFGR#126 
 1544330235 |   1996 | MFGR#127 
 1370946911 |   1996 | MFGR#128 
 1269919349 |   1996 | MFGR#129 
 1247033703 |   1997 | MFGR#121 
 1347066699 |   1997 | MFGR#1210
 1455475847 |   1997 | MFGR#1211
 1399803223 |   1997 | MFGR#1212
 1122484545 |   1997 | MFGR#1213
 1337347144 |   1997 | MFGR#1214
 1402515447 |   1997 | MFGR#1215
 1420052026 |   1997 | MFGR#1216
 1495082264 |   1997 | MFGR#1217
 1357824652 |   1997 | MFGR#1218
 1127355577 |   1997 | MFGR#1219
 1374061472 |   1997 | MFGR#122 
 1218324236 |   1997 | MFGR#1220
 1389236025 |   1997 | MFGR#1221
 1414147439 |   1997 | MFGR#1222
 1281336362 |   1997 | MFGR#1223
 1523832297 |   1997 | MFGR#1224
 1345236090 |   1997 | MFGR#1225
 1523314702 |   1997 | MFGR#1226
 1158340061 |   1997 | MFGR#1227
 1410752422 |   1997 | MFGR#1228
 1397754873 |   1997 | MFGR#1229
 1575447477 |   1997 | MFGR#123 
 1225023526 |   1997 | MFGR#1230
 1113962713 |   1997 | MFGR#1231
 1511260579 |   1997 | MFGR#1232
 1261364847 |   1997 | MFGR#1233
 1555658369 |   1997 | MFGR#1234
 1325601283 |   1997 | MFGR#1235
 1288704598 |   1997 | MFGR#1236
 1215812057 |   1997 | MFGR#1237
 1076279861 |   1997 | MFGR#1238
 1277424841 |   1997 | MFGR#1239
 1137540388 |   1997 | MFGR#124 
 1562295688 |   1997 | MFGR#1240
 1324107868 |   1997 | MFGR#125 
 1304429364 |   1997 | MFGR#126 
 1495219935 |   1997 | MFGR#127 
 1435838994 |   1997 | MFGR#128 
 1380654142 |   1997 | MFGR#129 
  728405195 |   1998 | MFGR#121 
  772241518 |   1998 | MFGR#1210
  654604431 |   1998 | MFGR#1211
  890208493 |   1998 | MFGR#1212
  640003860 |   1998 | MFGR#1213
  748112862 |   1998 | MFGR#1214
  737206930 |   1998 | MFGR#1215
  879344897 |   1998 | MFGR#1216
  797370310 |   1998 | MFGR#1217
  795668107 |   1998 | MFGR#1218
  660754584 |   1998 | MFGR#1219
  759924147 |   1998 | MFGR#122 
  776200672 |   1998 | MFGR#1220
  759235159 |   1998 | MFGR#1221
  858128275 |   1998 | MFGR#1222
  772868837 |   1998 | MFGR#1223
  827964147 |   1998 | MFGR#1224
  772631022 |   1998 | MFGR#1225
  934347224 |   1998 | MFGR#1226
  704757699 |   1998 | MFGR#1227
  719700730 |   1998 | MFGR#1228
  844398174 |   1998 | MFGR#1229
  741350529 |   1998 | MFGR#123 
  697727449 |   1998 | MFGR#1230
  715848717 |   1998 | MFGR#1231
  872721297 |   1998 | MFGR#1232
  698644386 |   1998 | MFGR#1233
  786058527 |   1998 | MFGR#1234
  711299651 |   1998 | MFGR#1235
  764901787 |   1998 | MFGR#1236
  689507081 |   1998 | MFGR#1237
  618842999 |   1998 | MFGR#1238
  854888093 |   1998 | MFGR#1239
  662699900 |   1998 | MFGR#124 
  937491980 |   1998 | MFGR#1240
  654370332 |   1998 | MFGR#125 
  798795601 |   1998 | MFGR#126 
  832030830 |   1998 | MFGR#127 
  787493701 |   1998 | MFGR#128 
  669040069 |   1998 | MFGR#129 
(280 rows)

